samples/application_development/code_relocation_nocopy/linker_arm_nocopy.ld

```diff
@@ -37,8 +37,8 @@
 /* On stm32 QSPI, external flash is mapped in XIP region at address given by the reg property. */
 
 #define EXTFLASH_NODE	DT_INST(0, st_stm32_qspi_nor)
-#define EXTFLASH_ADDR	DT_REG_ADDR(DT_INST(0, st_stm32_qspi_nor))
-#define EXTFLASH_SIZE	DT_REG_SIZE(DT_INST(0, st_stm32_qspi_nor))
+#define EXTFLASH_ADDR	DT_REG_ADDR_BY_IDX(DT_PARENT(EXTFLASH_NODE), 1)
+#define EXTFLASH_SIZE	DT_PROP(EXTFLASH_NODE, size) / 8
```

Comment:
Nitpicking: for long term maintenance, it would be better with outer parentheses (despite line 48 lacks of, and they do not seem to be mandated regarding current link files implementation).

Reply:
changed